sql/2024/fonts/design/fonts_family_by_script.sql

```diff
@@ -0,0 +1,47 @@
+-- Section: Design
+-- Question: Which families are used broken down by script?
+-- Normalization: Links (parsed only)
+
+-- INCLUDE ../common.sql
+
+WITH
+links AS (
+  SELECT
+    client,
+    SCRIPTS(payload) AS scripts,
+    FAMILY(payload) AS family,
+    COUNT(0) OVER (PARTITION BY client) AS total
+  FROM
+    `httparchive.all.requests`
+  WHERE
+    date = '2024-07-01' AND
+    type = 'font' AND
+    is_root_page AND
+    IS_PARSED(payload) AND
+    RAND() > 0.5
+)
+
+SELECT
+  client,
+  script,
+  family,
+  2 * COUNT(0) AS count,
+  2 * total AS total,
```

Comment:
Why? Can you add a comment to explain (same for line above).

Reply:
BigQuery was running out of resources, and I was downsampling a little to make it go through. I have just rerun the query without the downsampling, and it succeeded. Perhaps it was something temporary. I have removed this code now.